src/plugins/multimedia/_base.scss

```diff
@@ -74,6 +74,11 @@
 	vertical-align: middle;
 }
 
+%multmedia-details-summary-border-top-radius-0 {
+	border-top-left-radius: 0;
+	border-top-right-radius: 0;
+}
+
```

Comment:
This became a mixin very very specify, only for 1 use case and don't seems to be re-usable in other context.

Please just repeat those 2 line in your CSS code at line 191 and line 196.

Reply:
The placeholder selector is being extended twice. Are you sure you want me to eliminate it?